tutor/src/screens/question-library/exercises-display.jsx

```diff
@@ -203,6 +203,13 @@ class ExercisesDisplay extends React.Component {
         handler: this.onExerciseToggle,
       };
     }
+    if (!exercise.isMultiPart) {
+      const isUserGeneratedQuestion = exercise.belongsToCurrentUserProfileId(User.profile_id);
+      actions.copyEdit = {
```

Comment:
@camliu89 @jivewise we should add a `canEdit(user)` method to exercise that would check it it's not multipart and all this

Reply:
yep, that sounds good.